### ISSUE TITLE
GB->GB-NIR bug

### ISSUE THREAD

**Issue by corradio** [tmrowco/electricitymap#1472]
```
Traceback (most recent call last):
  File "/home/feeder/lib/fetch_data.py", line 130, in launch_parsers
    **parser_kwargs)
  File "/home/electricitymap/parsers/GB_NIR.py", line 303, in fetch_exchange
    exchange_dataframe = create_exchange_df(exchange_data)
  File "/home/electricitymap/parsers/GB_NIR.py", line 119, in create_exchange_df
    df_exchange = pd.read_csv(StringIO(text_data))
  File "/usr/local/lib/python3.6/site-packages/pandas/io/parsers.py", line 709, in parser_f
    return _read(filepath_or_buffer, kwds)
  File "/usr/local/lib/python3.6/site-packages/pandas/io/parsers.py", line 455, in _read
    data = parser.read(nrows)
  File "/usr/local/lib/python3.6/site-packages/pandas/io/parsers.py", line 1069, in read
    ret = self._engine.read(nrows)
  File "/usr/local/lib/python3.6/site-packages/pandas/io/parsers.py", line 1839, in read
    data = self._reader.read(nrows)
  File "pandas/_libs/parsers.pyx", line 902, in pandas._libs.parsers.TextReader.read
  File "pandas/_libs/parsers.pyx", line 924, in pandas._libs.parsers.TextReader._read_low_memory
  File "pandas/_libs/parsers.pyx", line 978, in pandas._libs.parsers.TextReader._read_rows
  File "pandas/_libs/parsers.pyx", line 965, in pandas._libs.parsers.TextReader._tokenize_rows
  File "pandas/_libs/parsers.pyx", line 2208, in pandas._libs.parsers.raise_parser_error
pandas.errors.ParserError: Error tokenizing data. C error: Expected 1 fields in line 39, saw 2
```

**Comment by corradio** [comment]
Also happens with GB-NIR->IE interconnector

**Comment by lorrieq** [comment]
fixed by #1506
